draft-mouris-cfrg-mastic.md

```diff
@@ -1281,14 +1277,37 @@ security analysis of Mastic is provided in {{MPDST25}}.
 
 # IANA Considerations
 
-TODO
+IANA is requested to add a new identifier to the "Verifiable Distributed
+Aggregation Functions (VDAF)" registry, that is described in {{Section 10 of
+!VDAF}}. The new requested entry to the VDAF Identifier registry is as follows:
+
+| Value         | Scheme                   | Type | Reference            |
+|:--------------|:-------------------------|:-----|:---------------------|
+| `0xFFFF0001`  | MasticCount              | VDAF | {{vdaf}} of RFC XXXX |
+| `0xFFFF0002`  | MasticSum                | VDAF | {{vdaf}} of RFC XXXX |
+| `0xFFFF0003`  | MasticSumVec             | VDAF | {{vdaf}} of RFC XXXX |
+| `0xFFFF0004`  | MasticHistogram          | VDAF | {{vdaf}} of RFC XXXX |
+| `0xFFFF0005`  | MasticMultihotCountVec   | VDAF | {{vdaf}} of RFC XXXX |
+{: #codepoints title="Additional codepoints for the VDAF Identifier Registry."}
+
+> TODO The codepoints in this section are from the private codepoint range and
+> are used for testing purposes. We will update this table with the codepoints
+> assigned by IANA.
+
+(RFC EDITOR: Please replace "RFC XXXX" above with the RFC number assigned to
+this document.)
 
 --- back
 
 # Acknowledgments
 {:numbered="false"}
 
-TODO
+The Network Error Logging and Attribute-Based Browser Telemetry applications
```

Comment:
These applications aren't described until the appendix. At a minimum we need a forward reference here, but I would just cut it.

It looks like this was copied directly from the paper. Note that the ACK section of an IETF draft doesn't necessarily serve the same purpose as in a paper. I would rather acknowledge folks who have contributed to the draft, e.g., Pratik Sarkar and David Cook.

Acknowledging a funding source is definitely acceptable, but I'm not sure Dimitris is supported by NSF anymore :)

Reply:
Revised the reference to the Appendix/Motivation section to make it clear. We have Pratik and David as contributors (line 45) -- I think this is OK. RE: NSF support, I think it makes sense to Ack if this was the case before.